lib/ansible/plugins/become/__init__.py

```diff
@@ -39,6 +39,10 @@ def __init__(self):
         self._id = ''
         self.success = ''
 
+    def set_options(self, *args, **kwargs):
+        super(BecomeBase, self).set_options(*args, **kwargs)
+        self.prompt = ''
```

Comment:
I wonder though if we should just do it here: https://github.com/ansible/ansible/blob/20c4ad3b95ff7b35c36737f85ed600afd5628bea/lib/ansible/executor/task_executor.py#L413

since the issue is related to loops?